fission/src/systems/scene/SceneRenderer.ts

```diff
@@ -68,11 +73,34 @@ class SceneRenderer extends WorldSystem {
         const ambientLight = new THREE.AmbientLight(0xffffff, 0.1)
         this._scene.add(ambientLight)
 
-        const ground = new THREE.Mesh(new THREE.BoxGeometry(10, 1, 10), this.CreateToonMaterial(GROUND_COLOR))
-        ground.position.set(0.0, -2.0, 0.0)
-        ground.receiveShadow = true
-        ground.castShadow = true
-        this._scene.add(ground)
+        const ground = new THREE.Mesh(new THREE.BoxGeometry(10, 1, 10), this.CreateToonMaterial(GROUND_COLOR));
+        ground.position.set(0.0, -2.0, 0.0);
+        ground.receiveShadow = true;
+        ground.castShadow = true;
+        this._scene.add(ground);
+
+        // skybox
+        const currentTheme = (window as any).getTheme();
```

Comment:
ESLint wants a different type here instead of `any`. Any chance we can change this?

Reply:
@Dhruv-0-Arora What is this `getTheme()` function?

Reply:
If you're trying to get the background to update to the scene, I would build out a component that, when re-rendered, updates the uniforms of the skybox

Reply:
the get theme function is supposed to get the currentTheme. Right now its just black

Reply:
I think I probably need to fix it